apps/logo-quiz/src/environments/environment.interface.ts

```diff
@@ -1,3 +1,4 @@
 export interface Environment {
```

Comment:
why is this file duplicated?

Reply:
There's one for the `api` app and for the `logo-quiz` app. This change moves the file around because I didn't Nx had already configured React with environments.